### PR TITLE
6.2 - Implement SignInServlet

### PR DESCRIPTION
**Pull request by Bonoparte**

### CODE REVIEW

File: src/main/java/servlets/SignInServlet.java [hunk 1]
```diff
@@ -10,38 +9,66 @@
...
+			req.getRequestDispatcher("/adminpage.jsp").forward(req, resp);
 		}
 	}
-	
+
+
 	private String checkUser(String login, String password) {
-		String      result      = "not banned";
-		UserService userService = new UserService(login,password);
+		String result = "active";
```
Comment (MrToady) [inline src/main/java/servlets/SignInServlet.java]: this variable is redundant

File: src/main/java/servlets/SignInServlet.java [hunk 2]
```diff
@@ -10,38 +9,66 @@
...
+	protected void doGet(HttpServletRequest req, HttpServletResponse resp)
 			throws ServletException, IOException {
-		doPost(req, resp);
+		HttpSession session = req.getSession();
+		String login;
+		String password;
+		login = (String) session.getAttribute("login");
+		password = (String) session.getAttribute("password");
+		if (login != null && password != null) {
+			if (login.equalsIgnoreCase("admin")) {
```
Comment (acorned) [inline src/main/java/servlets/SignInServlet.java]: Will we have only one admin account in our system?

File: src/main/java/servlets/SignInServlet.java [hunk 3]
```diff
@@ -10,38 +9,66 @@
...
+		String password;
+		login = (String) req.getParameter("login");
+		password = (String) req.getParameter("password");
+		//TODO: logic of verification login and password in DB
+		//TODO: logic of checking login's status
+		checker = "active";
+		if (login.equals("admin")) {
+			checker = "admin";
+		}
+		if (login.equals("banned")) {
```
Comment (acorned) [inline src/main/java/servlets/SignInServlet.java]: I thought that 'banned' is a status, not login.

File: src/main/java/servlets/SignInServlet.java [hunk 4]
```diff
@@ -10,38 +9,66 @@
...
+			checker = "banned";
+		}
```
Comment (acorned) [inline src/main/java/servlets/SignInServlet.java]: Keep it more simple, like this:
checker = (condition) ? "admin" : (condition) ? "banned" : active;

**Review by MrToady — APPROVED** [review]

**Pull request merged**